libs/ui-lib/lib/cim/components/Agent/BMCForm.tsx

```diff
@@ -228,8 +231,13 @@ const getInitValues = (
       macMapping: nmState?.spec?.interfaces || [{ macAddress: '', name: '' }],
     };
   } else {
-    return emptyValues;
+    values = emptyValues;
+  }
+
+  if (!addNMState) {
+    values.nmState = '';
```

Comment:
It's probably ok for recent flow, but the `addNMState` and `isEdit` are theoretically independent on each other and so the `values` can be `undefined` here.

What about changing the line above to:
```
let values = emptyValues;
```